config/initializers/cardinal_config.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'csv'
+cardinal_pooling_csv = CSV.read('config/cardinal_pooling.csv', { encoding: 'UTF-8', headers: true, header_converters: :symbol, converters: :all })
+
+Rails.application.config.cardinal_pooling_config = {}
+
+cardinal_pooling_csv.each do |data|
+  contents = data.to_hash
+  number_of_pools = data.to_hash.values.compact.count - 1
+
+  Rails.application.config.cardinal_pooling_config[contents[:number]] = number_of_pools
```

Comment:
Comment from Stuart: 

Those pooling strategies look a lot better than before.  They actually look like you could write a more general rule than having a config file.
Something like, upto 20 samples is a single pool, then multiples of 13 are the transition from x pools to x+1 pool upto 76 samples where it changes to multiples of 11.  

Todo: check whether there should be more than 13 samples in a pool, and that the CSV is correct

Reply:
Going to leave this for now as the CSV makes it clear how many pool are allocated for a number of passed samples